xarray_user_forum_2021/Xarray_User_Summit.ipynb

```diff
@@ -0,0 +1,8915 @@
+{
```

Comment:
<p>Also Dask! This is still wip but parts of ArviZ are already compatible with Dask backed xarray objects, and as a thin wrapper, InferenceData itself already is. Probably important to note that this feature is already being used, even in production!</p><p> </p><p>Some example cases where this has been useful are regression-like cases where # params &lt;&lt; # observations so things like pointwise log likelihood, posterior predictive and so on with sizes of # samples * # observations don't fit to memory anymore but are still useful for a Bayesian workflow, or cases where we have N parameters but we are actually interested in M&gt;&gt;N different params/effects so that N * # samples fits in memory but M * # samples doesn't anymore.</p><p> </p><p>Also tangentially related: https://docs.dask.org/en/latest/array-best-practices.html#consider-xarray. With ArviZ, people doing Bayesian things is generally exposed first to xarray and can use dask if needed with little to no overhead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_misc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Added a slide